### ISSUE TITLE
Load failed: TypeError: Unable to get property 'inputs' of undefined or null reference

### ISSUE THREAD

**Issue by lgtanimoto** [jmoenig/Snap--Build-Your-Own-Blocks#1645]
When I try to open the below SNAP! link

http://snap.berkeley.edu/snapsource/snap.html#present:Username=kchinds8&ProjectName=cowboy

I get the error "Load failed: TypeError: Unable to get property 'inputs' of undefined or null reference.

This project was created by a student of mine.   It was working fine until she tried configuring the language options to Spanish.   Even changing language back does not help.

I saw some similar problems in the forum that needed admin support to repair.  Is this the case here?

Regards,

**Comment by jmoenig** [comment]
Looking into this...

**Comment by jmoenig** [comment]
I'm sorry about this, I hope your student doesn't feel discouraged from working on her project, I'll do everything I can to help. I did manage to repair the project. She can open the repaired version here:

http://snap.berkeley.edu/snapsource/snap.html#cloud:Username=jens&ProjectName=cowboy

and just save it again to her account and keep working on it. This should do the trick.

I also introduced a fix that prevents this from happening again in future versions. However, since this fix is not yet officially released there is a slight chance that this might happen again. In this case your student can open her project in the current development version:

http://snap.berkeley.edu/snapsource/dev/snap.html

she can also use this link to directly open it in the dev version:

http://snap.berkeley.edu/snapsource/dev/snap.html#cloud:Username=kchinds8&ProjectName=cowboy

This should always works, even if the project cannot be loaded from the official url, and then simply save it back to the cloud. After this she should be able to re-open the project from the regular url again.

That said, I'm not entirely sure how this happened - it's clearly not the student's fault, though, and switching to another language didn't cause it -.
